recipes/libcap/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+import os
+import glob
+
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+
+
+class LibcapConan(ConanFile):
+    name = "libcap"
+    license = ("GPL-2.0", "BSD-3-Clause")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://git.kernel.org/pub/scm/libs/libcap/libcap.git"
+    description = "This is a library for getting and setting POSIX.1e" \
+                  " (formerly POSIX 6) draft 15 capabilities"
+    topics = ("conan", "libcap", "capabilities")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "psx_syscals": [True, False]}
```

Comment:
It will require a patch, but you can add the `fPIC` option by patching out `-fPIC` at this line:
https://git.kernel.org/pub/scm/libs/libcap/libcap.git/tree/Make.Rules#n55
+ make sure that `-fPIC` is added to `CFLAGS/LDFLAGS` when compilng/linking the library.

Reply:
I checked that CFLAGS/LDFLAGS defined by conan is properly handled by makefiles. And looks like fPIC option is working now.